### ISSUE TITLE
Web UI font does not have symbol for ⁻ (superscript minus)

### ISSUE THREAD

**Issue by septatrix** [sharkdp/numbat#660]
They are display as missing chars instead

**Comment by sharkdp** [comment]
It seems to work fine for me?

![image](https://github.com/user-attachments/assets/d43faeb7-3015-45a0-886b-385b8380404d)

**Comment by septatrix** [comment]
Likely because your fallback font for `monospace` has that font available. For me this happens on Chrome/Android whereas on my desktop the glyph is displayed because "Noto Sans Mono" has it. In the Firefox and Chrome devtools it even shows that Noto Sans Mono provides one glyph when it includes a superscript minus. You can also check on Google fonts that Fira Mono does not provide the superscript minus glyph. Adding another fallback is likely sufficient as a workaround (though I also raised this issue upstream: https://github.com/mozilla/Fira/issues/253)

**Comment by sharkdp** [comment]
Thank you for the explanation!

Adding another fallback would probably mean we would have to load an additional font from https://fonts.google.com/?

**Comment by septatrix** [comment]
> Adding another fallback would probably mean we would have to load an additional font from https://fonts.google.com/?

Yes though I think that it would not be loaded unless required but I could be mistaken
